examples/multi_platform/MixedLib/BUILD

```diff
@@ -23,6 +48,19 @@ swift_library(
     deps = [":MixedAnswer"],
 )
 
+objc_library(
+    name = "ObjcLibDependingOnMixedLib",
+    srcs = [
+        "ObjcLibDependingOnMixedLib.m",
+    ],
+    hdrs = [
+        "ObjcLibDependingOnMixedLib.h",
+    ],
+    copts = ["-I."],
```

Comment:
Same here WRT an explanation of this existing in comment form.

Reply:
Added